### ISSUE TITLE
How to persist datasets / analytics across Docker restarts?

### ISSUE THREAD

**Issue by sooheon** [bigomics/omicsplayground#109]
Not familiar with R or Shiny, but I assume there are ways to persist state (datasets, analysis outputs) to local storage. Is there a documented way to do this so that results are not lost if the Docker container is restarted?

**Comment by ivokwee** [comment]
Yes you can. It's not a R/Shiny but as you pointed out more of a Docker issue. Just launch the Docker with 

`-v /home/me/mydata:/omicplayground/data`

Then the computed dataset will be stored and ready next time.

**Comment by sooheon** [comment]
Great, thanks!
